src/praisonai-agents/tests/unit/test_gateway_turn_executor.py

```
@@ -0,0 +1,79 @@
+def test_inprocess_executor_protocol_conformance():
+    assert isinstance(InProcessTurnExecutor(), TurnExecutorProtocol)
+
+
+def test_worker_wedged_error_is_exception():
+    assert issubclass(WorkerWedgedError, Exception)
+
+
+def test_turn_placement_is_frozen_value_type():
+    p = TurnPlacement(session_id="s", worker_id="w", epoch=1)
+    assert p == TurnPlacement(session_id="s", worker_id="w", epoch=1)
+    with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)